src/trace_processor/perfetto_sql/stdlib/android/memory/dmabuf.sql

```diff
@@ -144,3 +146,24 @@ LEFT JOIN process
   USING (upid)
 ORDER BY
   ts;
+
+-- Provides a timeseries of dmabuf allocations for each process.
+-- To populate this table, tracing must be enabled with the "dmabuf_allocs" ftrace event.
+CREATE PERFETTO TABLE android_memory_cumulative_dmabuf (
+  -- upid of process responsible for the allocation (matches utid)
```

Comment:
Would suggest exposing the thread name and process name here as well using left joins to make this table more useful to query.